server/lib/nl/common/constants.py

```diff
@@ -327,6 +327,8 @@
         "Household Median Income",
     "Median_Earnings_Person":
         "Individual Median Earnings",
+    "dc/e9gftzl2hm8h9":
+        "Total Commute Time Across All Workers",
```

Comment:
Given the description that immediately follows it, and the other shorter titles, tempted to say, keep it short?  esp the "Across All Workers" part.

Reply:
Sounds good. Shortened to "Total Commute Time" while keeping the description the same.